exoctk/exoctk_app/app_exoctk.py

```diff
@@ -300,7 +300,14 @@ def groups_integrations():
                 kmag = data.get('Kmag')
 
                 # Transit duration in exomast is in days, need it in hours
-                obs_time = data.get('transit_duration')*u.Unit(form.time_unit.data).to('hour')
+                if form.time_unit.data == 'day':
+                    obs_dur = data.get('transit_duration')
+                    form.obs_duration.data = obs_dur
+                else:
+                    trans_dur = data.get('transit_duration')
+                    trans_dur *= u.Unit('day').to('hour')
+                    obs_dur = 3*trans_dur + 1
+                    form.obs_duration.data = obs_dur
```

Comment:
@mfixstsci Why is there logic `3*trans_dur + 1` that only happens if it's in days? Shouldn't this be the same conversion days or hours?

Reply:
@kbs said this the calculation to get a rough estimate of the observation time based on the transit duration converted to hours. I believe that he mentioned it has something to do with the JWST readouts in-between observations and that there is an hour lag time on the beginning and end of an observation. I am still unclear why the transit duration in days maps 1-to-1 to the observation time but there is a conversion required if it is in hours. Or maybe there needs to be a correction applied to both.

Reply:
@kevin218 please weigh in?

Reply:
Yes, Mees is correct:  `obs_dur = 3*trans_dur + 1`

Reply:
@mfixstsci It should apply for both days and hours -- can you update as such?

Reply:
Okay, will do! Working on it now, thanks @kevin218 and @julesfowler

Reply:
@kevin218 does this only apply to the `+ 1`?

Reply:
@mfixstsci Oooh, if we want 30 minutes of overhead on either side I would guess it's `3*trans_dur + 1 hour`?

Reply:
@julesfowler Good point!  Yes to `3*trans_dur + 1 hour`

Reply:
@julesfowler makes sense, I figured the 3 was just to scale the calculation. In days you'd want `3*trans_dur + (1/24.)`

Reply:
@mfixstsci Yes, use `3*trans_dur + (1/24.)` when in units of days.